From 03eecb053e1137a7a175c7041ba571ddc815d2b7 Mon Sep 17 00:00:00 2001
From: Alan Garcia <alan.garcia@continental-corporation.com>
Date: Thu, 21 Nov 2013 11:06:34 -0600
Subject: [PATCH 08/15] Reduce systemd timeout values

Default timeout values used by systemd to control processes were reduced
to avoid wait much time for its completion.

Ticket-Id: BZ553
Change-Id: I4bc2c9174e032e304cf8c56831eae04bf6977c56
---
 src/shared/def.h |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/shared/def.h b/src/shared/def.h
index 5abb544..6deab68 100644
--- a/src/shared/def.h
+++ b/src/shared/def.h
@@ -23,11 +23,11 @@
 
 #include "util.h"
 
-#define DEFAULT_TIMEOUT_USEC (90*USEC_PER_SEC)
+#define DEFAULT_TIMEOUT_USEC (10*USEC_PER_SEC)
 #define DEFAULT_RESTART_USEC (100*USEC_PER_MSEC)
-#define DEFAULT_CONFIRM_USEC (30*USEC_PER_SEC)
+#define DEFAULT_CONFIRM_USEC (10*USEC_PER_SEC)
 
-#define DEFAULT_EXIT_USEC (5*USEC_PER_MINUTE)
+#define DEFAULT_EXIT_USEC (1*USEC_PER_MINUTE)
 
 #define SYSTEMD_CGROUP_CONTROLLER "name=systemd"
 
-- 
1.7.9.5

